WordPress/src/test/java/org/wordpress/android/ui/jetpack/JetpackCapabilitiesUseCaseTest.kt

```diff
@@ -0,0 +1,112 @@
+package org.wordpress.android.ui.jetpack
+
```

Comment:
Same as above - I renamed the file and I'm not sure why github is treating it as remove and add.